core/src/main/java/org/mule/context/notification/DefaultFlowCallStack.java

```diff
@@ -19,69 +17,83 @@
  * Keeps context information about the executing flows and its callers
  * in order to provide augmented troubleshooting information for an application developer.
  */
-public class DefaultFlowCallStack implements FlowCallStack, Serializable
+public class DefaultFlowCallStack implements FlowCallStack
 {
-    private static final long serialVersionUID = -4540003678773678613L;
+    private static final long serialVersionUID = -8683711977929802819L;
 
-    private Stack<FlowStackElement> innerStack = new Stack<>();
+    private Stack<DefaultFlowStackElement> innerStack = new Stack<>();
 
-    public DefaultFlowCallStack()
+    /**
+     * Adds an element to the top of this stack
+     * 
+     * @param flowStackElement the element to add
+     */
+    public void push(DefaultFlowStackElement flowStackElement)
     {
+        innerStack.push(flowStackElement);
     }
 
-    public DefaultFlowCallStack(FlowCallStack flowCallStack)
+    /**
+     * Adds a message processor path to the list of processors that were invoked as part of the processing of this stack's event.
```

Comment:
Use javadoc lines formatted to 100/120 characters, the text is easier to read as you don't need to scroll